contentcuration/contentcuration/frontend/channelEdit/components/edit/EditModal.vue

```diff
@@ -534,11 +534,17 @@
           return newNodeId;
         });
       },
+      resetInheritMetadataModal() {
+        this.$refs.inheritModal?.checkInheritance();
```

Comment:
Just noticed that just reseting the closed state wasnt inheriting the fields that were already marked as not ask again 😅, this fixes this checkInheritance fixes this!